test/app/background/api/cosmeticFilterAPITest.ts

```diff
@@ -0,0 +1,263 @@
+
+    before(function () {
+      this.getStorageStub = sinon.stub(chrome.storage.local, 'get')
+      this.setStorageStub = sinon.stub(chrome.storage.local, 'set').callsFake(function (value) {
+        sampleValue = value
+      })
+    })
+    after(function () {
+      this.getStorageStub.restore()
+      this.setStorageStub.restore()
+    })
+    beforeEach(function () {
+      this.getStorageStub.reset()
+      this.setStorageStub.reset()
+    })
+    it('passes only 1 arg to chrome.storage.local.set', function () {
+      this.getStorageStub.yields({
+        'list': {
+          url: filter
+        }
+      })
+      cosmeticFilterAPI.removeSiteFilter(url)
+      assert.equal(this.setStorageStub.getCall(0).args.length, 1)
+    })
+    it('removes the correct filter', function () {
+      this.getStorageStub.yields({
+        list: {
+          'https://www.brave.com': ['#cssFilter'],
+          'https://notbrave.com': ['notACSSFilter']
+        }
+      })
+      cosmeticFilterAPI.removeSiteFilter(url)
+      assert.deepEqual(this.setStorageStub.getCall(0).args[0], {
+        list: {
+          'https://notbrave.com': ['notACSSFilter']
+        }
+      })
+    })
+    it('handles empty storage', function () {
+      this.getStorageStub.yields({})
+      cosmeticFilterAPI.removeSiteFilter(url)
+      assert.deepEqual(this.setStorageStub.getCall(0).args[0], {
+        list: {}
+      })
+    })
+    it('handles undefined storage', function () {
+      this.getStorageStub.yields(undefined)
+      cosmeticFilterAPI.removeSiteFilter(url)
+      assert.deepEqual(this.setStorageStub.getCall(0).args[0], {
+        list: {}
+      })
+    })
+    it('handles url not in storage', function () {
+      this.getStorageStub.yields({
+        list: {
+          url: filter
+        }
+      })
+      cosmeticFilterAPI.removeSiteFilter('urlNotInStorage')
+      assert.deepEqual(this.setStorageStub.getCall(0).args[0], {
+        list: {
+          url: filter
+        }
+      })
+    })
+  })
+  describe('removeAllFilters', function () {
+    before(function () {
+      this.getStorageStub = sinon.stub(chrome.storage.local, 'get')
+      this.setStorageStub = sinon.stub(chrome.storage.local, 'set')
+    })
+    after(function () {
+      this.getStorageStub.restore()
+      this.setStorageStub.restore()
+    })
+    beforeEach(function () {
+      this.getStorageStub.reset()
+      this.setStorageStub.reset()
+    })
+
+    it('sets empty list object', function () {
+      this.getStorageStub.yields({
+        list: {
+          'hostname': 'isNotEmpty'
+        }
+      })
+      cosmeticFilterAPI.removeAllFilters()
+      assert.deepEqual(this.setStorageStub.getCall(0).args[0], {
+        list: {}
+      })
+    })
+  })
+  describe('applySiteFilters', function () {
+    const filter = '#cssFilter'
+    const filter2 = '#cssFilter2'
+
+    let sampleValue: any
+    let CSSvalue: any
+
+    console.log(sampleValue, CSSvalue)
```

Comment:
ditto for console methods

Reply:
`sampleValue` and `CSSvalue` have to be explicitly called, see above (is there any way to do this without doing a `console.log`?